README.md

```diff
@@ -1,3 +1,38 @@
 # Cookbook [![Build Status](https://travis-ci.org/CodemateLtd/cookbook.svg?branch=master)](https://travis-ci.org/CodemateLtd/cookbook)
 
 > Recipes and best practices for awesome digital teams.
+
+```txt
+      __...--~~~~~-._   _.-~~~~~--...__
+    //   Vorsprung   `V'    Pauca      \\
+   //   durch Technik |     sed bona    \\
+  //__...--~~~~~~-._  |  _.-~~~~~~--...__\\
+ //__.....----~~~~._\ | /_.~~~~----.....__\\
+====================\\|//====================
+                    `---`
+```
+
+## What is this
+
+This repository contains all of the recipes, guidelines, templates, documentation you can reuse to cook a delicious digital project. Think of it as a reference of practices to use when working as a team; a starting point to define your own team’s practices.
+
+This cookbook is about helping **teams** succeed first and foremost, projects being successful is only a consequence. **It defines where we want to be**, not necessarily where we always are. It is meant to apply to **all teams, and all team members** – we are knowledge workers first and foremost, our most important role is to successfully think, organise, and share knowledge. Code, designs, plans are only specific artifacts.
+
+## Contents
+
+*   **Guidelines**
+    *   Contribution guidelines: [`CONTRIBUTING.md`](CONTRIBUTING.md)
+        *   Covers branching, and code reviews.
+    *   Pull request template: [`.github/PULL_REQUEST_TEMPLATE.md`](.github/PULL_REQUEST_TEMPLATE.md)
```

Comment:
This is really nice. :+1: 

Would be great to also add information how to do templates with Gitlab. Basically just link to this: https://docs.gitlab.com/ee/user/project/description_templates.html#creating-merge-request-templates and same markdown format should work just fine.

Tried to do some quick research if those could be added as organization (or hosted instance in Gitlab case) defaults instead of creating per repository basis, but didn't find any clue.

Reply:
👍 will add the link